Base/usr/share/man/man1/syscall.md

```diff
@@ -0,0 +1,58 @@
+## Name
+
+syscall - test a system call
+
+## Synopsis
+
+`syscall [-o] [-l] [-h] entry [args; buf==BUFSIZ buffer]`
+
+## Description
+
+syscall(1) is an effort to port the plan9 syscall command to Serenity. It can invoke a system call with the given arguments from the command line.
+
+## Options
+
+* `-o`: Output to stdout the contents of the buffer argument specified as buf.
+* `-l`: Print all the serenity system calls and exit. Note that not all the system calls can be invoked using this tool.
```

Comment:
"Serenity" (capitalize it)

Given this it's not clear *what* does it print about those calls — their names, one per line, I would guess?